homeassistant/data_entry_flow.py

```diff
@@ -134,30 +135,9 @@ class FlowResult(TypedDict, total=False):
     title: str
     type: FlowResultType
```

Comment:
Also wondering, should we make a ConfigFlowResultType? Because CREATE_ENTRY is config flow specific

Reply:
Yes, that makes sense 👍